packages/core-mobile/app/new/common/components/CollapsibleTabs.tsx

```diff
@@ -75,15 +77,6 @@ const CollapsibleTabWrapper = ({
     }
   )
 
-  useAnimatedReaction(
-    () => tabIndex.value,
-    (curr, prev) => {
-      if (curr !== prev && onScrollTab) {
-        runOnJS(onScrollTab)(tabIndex)
-      }
-    }
-  )
-
```

Comment:
@onghwan this doesn't seem to be needed so I removed it. it was doing a lot of work on the js thread.